### ISSUE TITLE
With the latest "next" snapshots, the dialog for no new available updates is not centered on the screen

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#8092]
### Steps to reproduce:
1. Install the latest "next" snapshot of NVDA, insuring that there are no new updates available.
2. Check for updates to NVDA from NVDA menu -> Help -> Check for updates.
3. When the dialog informing you that there are no new updates appears, check its position on the screen.

### Expected behavior:
NVDA's dialog for no new updates should be centered on the screen.

### Actual behavior:
NVDA's dialog for no new updates is positioned in the top left corner of the screen.

### System configuration:
- Operating system: Windows 10 Pro version 1709 (build 16299.309), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-14952,dd1cbe50.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

### Additional information:
This issue may apply for some other dialogs: the one for available updates (with the three buttons) and the one for installing now or postponing an update) as well, but I hadn't got the chance to test it yet.

**Comment by Brian1Gaff** [comment]
I can think of at least one other program that does this, or used to 
ccleaner!
 However you are right in that it used to be in the centre, but I've not 
checked it recently.
 Brian

**Comment by leonardder** [comment]
Ough, my fault. It turns out that we have wx.CENTER and wx.Center. I used the former, where it should have been the latter. Life's so confusing!

**Comment by leonardder** [comment]
See https://github.com/nvaccess/nvda/pull/7104#issuecomment-373400247 for a more in depth explanation of what's going wrong here.

**Comment by leonardder** [comment]
I believe that this has been fixed. Please request a reopen if that's not the case.
